### PR TITLE
Update csi sidecar versions on staging rc

### PR DESCRIPTION
**Pull request by Jiawei0227**
Upgrade csi sidecar to the latest version except csi-snapshotter.

### CODE REVIEW

File: deploy/kubernetes/base/controller/csidriver_info.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
-#### TODO: This need to be updated to v1 once we official support 1.18+
-apiVersion: storage.k8s.io/v1beta1
```
Comment (mattcary) [inline deploy/kubernetes/base/controller/csidriver_info.yaml:2]: We say that dev supports 1.17+, so we can't move to v1 csi driver yet.

Comment (Jiawei0227) [reply deploy/kubernetes/base/controller/csidriver_info.yaml:2]: sure let me revert it.

Comment (Jiawei0227) [reply deploy/kubernetes/base/controller/csidriver_info.yaml]: Done.

**Comment by mattcary** [comment]
/lgtm

thanks!

**Pull request merged**